components/brave_rewards/resources/rewards_panel/lib/interfaces.ts

```diff
@@ -33,6 +34,8 @@ export interface HostState {
   publisherInfo: PublisherInfo | null
   externalWallet: ExternalWallet | null
   summaryData: RewardsSummaryData
+  notifications: Notification[]
+  notificationsLastViewed: number
```

Comment:
Should this be a `Date` or something similar?

Reply:
I've started to move toward using `number` for application state that represents dates, for a few reasons:

* It's straightforward to serialize to JSON and across boundaries (extension APIs, webUI message handlers, etc.)
* The JS Date API is kinda not great - it's often easier to use time numbers and only bring in `Date` when necessary.
* It's easier to get confused about local/UTC time when using `Date`.

Reply:
OK, makes sense.